dbt/README.md

```diff
@@ -96,27 +243,90 @@ dbt docs serve
 
 Then, navigate to http://localhost:8080 to view the site.
 
-### Run tests
+<h3 id="how-to-add-a-new-model"> ➕ How to add a new model </h3>
 
-Run the tests:
+To request the addition of a new model, open an issue using the [Add a new dbt
+model](.github/ISSUE_TEMPLATE/new-dbt-model.md) issue template. The assignee
+should follow the checklist in the body of the issue in order to add the model
+to the DAG.
 
-```
-dbt test
-```
+There are a few subtleties to consider when requesting a new model, outlined
+below.
 
-Run tests for only one model:
+#### Model materialization
 
-```
-dbt test --select <model_name>
-```
+There are a number of different ways of materializing tables in Athena
+using dbt; see the [dbt
+docs](https://docs.getdbt.com/docs/build/materializations) for more
+detail.
 
-Run tests for dbt macros:
+So far our DAG only uses view and table materialization, although we are
+interested in eventually incorporating incremental materialization as well.
 
-```
-dbt run-operation test_all
-```
+The choice between view and table materialization depends on the runtime and
+downstream consumption of the model query. There is no hard and fast rule, but
+as a general guideline, consider materializing a model as a table if queries
+using the table take longer than 30s to execute, or if the model is consumed
+by another model that is itself computationally intensive.
+
+Our DAG is configured to materialize models as views by default, so
+extra configuration is only required for non-view materialization.
+
+#### Model naming
+
+Models should be namespaced according to the database that the model lives in
+(e.g. `location.tax` for the `tax` table in the `location` database.) Since
+dbt does not yet support namespacing for refs, we include the database as a
+prefix in the model name to simulate real namespacing, and we override the
+`generate_alias_name` macro to strip out this fake namespace when generating
+names of tables and views in Athena
+([docs](https://docs.getdbt.com/docs/build/custom-aliases)).
 
-#### Debugging dbt test failures
+In addition to database namespacing, views should be named with a `vw_` prefix
+(e.g. `location.vw_pin10_location`) to mark them as a view, while tables do not
+require any prefix (e.g. `location.tax`).
+
+#### Model description
+
+All new models should include, at minimum, a
+[description](https://docs.getdbt.com/reference/resource-properties/description)
+of the model itself. We generally store these descriptions as [docs
+blocks](https://docs.getdbt.com/reference/resource-properties/description#use-a-docs-block-in-a-description).
+
+Ideally, new models should also include descriptions for each column,
+implemented directly in the `schema.yml` model definition. However, our
+column descriptions are currently sparse, so it is not a strict requirement just
+yet.
```

Comment:
**suggestion:** I think we should just make it a requirement for new models, since this will be changed in the near future.

Reply:
Good call, done in 7426ff0c17694b32f931889a6d6082b6b0bb049b!